src/ssl.ml

```diff
@@ -77,6 +77,26 @@ type verify_error =
   | Error_v_keyusage_no_certsign
   | Error_v_application_verification
 
+module Modes = struct
+  type t = int
+
+  let no_mode                    = 0x000
+  let enable_partial_write       = 0x001
+  (*let accept_moving_write_buffer = 0x002: is always set because of GC*)
+  let auto_retry                 = 0x004
+  let no_auto_chain              = 0x008
+  let release_buffers            = 0x010
+  let send_clienthello_time      = 0x020
+  let send_serverhello_time      = 0x040
+  let send_fallback_scsv         = 0x080
+  let async                      = 0x100
+
+  let (lor) = (lor)
+end
+
+  (** Allow SSL_write(..., n) to return r with 0 < r < n (i.e. report success
```

Comment:
Is this comment misplaced? What's doing this exactly?

Reply:
Yeh, a wrong paste: fixed